### PR TITLE
Usage information

### PR DESCRIPTION
**Pull request by benediktahrens**
- add a file on usage of UniMath
- move info on coqide to separate file
- make INSTALL more readable by bulleting

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+During the [installation procedure](./INSTALL.md) you have set up ProofGeneral on your computer.
+ProofGeneral is an add-on to the text editor Emacs, adding buttons, menus, and keyboard shortcuts
+to interact with Coq, the proof assistant that UniMath relies on.
+
+Upon opening a Coq file (i.e., a file with ending *.v) in Emacs, the ProofGeneral add-on is automatically
+loaded by Emacs.
+Andrej Bauer has a [screencast](https://www.youtube.com/watch?v=l6zqLJQCnzo) on using ProofGeneral
+(more screencasts are listed on his [website](http://math.andrej.com/2011/02/22/video-tutorials-for-the-coq-proof-assistant/)).
+Note that for Bauer's specific example to work in UniMath, you need to insert the line
```
Comment (DanGrayson) [inline USAGE.md]: Maybe say that ProofGeneral doesn't load the prelude only if you are stepping through I file located in our UniMath directory tree.  It's also possible to use ProofGeneral outside that tree, if Coq is installed on the system.

File: USAGE.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+at the beginning of the file, since UniMath does not load this library by default.
+
+*NB:* Emacs loads special settings to handle UniMath files. These settings are only loaded when
+editing files in the subdirectory `UniMath/UniMath`.
+To edit your own files with the same comfort, place them in this subdirectory.
+
+Unicode input
+-------------
+Unicode symbols are used throughout UniMath. To see how to input a specific Unicode character, type
+`M-x describe-char` while hovering over that character.
```
Comment (DanGrayson) [inline USAGE.md]: Maybe say that function is also available on C-u C-x =

File: USAGE.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| Mathematical symbol | UniMath Unicode |UniMath ASCII |
+|---------------------|-----------------|-----------------|
+|  ∏ (x : A), B       | `\prod`          | `forall x : A, B`      |
+| A → B               | `\to`            | `A -> B`  |
+|  λ x, e             | `\lambda`          | `fun x => e`  |
+| ∑ (x : A), B        | `\sum`         | `total2 (fun x => B)` |
+|  X × Y              | `\times`       | `dirprod X Y` |
+|  (a,b)              |                | `a,,b`|
+| X ⨿ Y               | `C-X 8 RET AMALGAMATION OR COPRODUCT`| `coprod`|
+| a = b               |             | `=`  |
```
Comment (DanGrayson) [inline USAGE.md]: For consistency, the UniMath ASCII can be a=b here/

**Comment by DanGrayson** [comment]
Maybe now is a good time to add instructions about using the standard Coq installed on the system, instead of compiling the submodule.  This is done with BUILD_COQ=no, and can be set permanently following the instructions in build/Makefile-configuration-template

**Comment by benediktahrens** [comment]
On 03/03/2019 08:59, Daniel R. Grayson wrote:
> Maybe now is a good time to add instructions about using the standard 
> Coq installed on the system, instead of compiling the submodule. This is 
> done with BUILD_COQ=no, and can be set permanently following the 
> instructions in build/Makefile-configuration-template

Are we currently testing against a standard Coq, i.e., against a 
released version?

**Comment by benediktahrens** [comment]
Closing this temporarily...

**Comment by siddharthist** [comment]
I have only ever used standard versions of Cos, it's worked just fine for me.

**Comment by DanGrayson** [comment]
> On 03/03/2019 08:59, Daniel R. Grayson wrote: Maybe now is a good time to add instructions about using the standard Coq installed on the system, instead of compiling the submodule. This is done with BUILD_COQ=no, and can be set permanently following the instructions in build/Makefile-configuration-template
> Are we currently testing against a standard Coq, i.e., against a released version?

Yes.  Well, you're usually the one who updates it when a new standard version comes out, and sub/coq is always pointing to the latest standard version.

**Comment by benediktahrens** [comment]
On 03/03/2019 18:06, Daniel R. Grayson wrote:
>     On 03/03/2019 08:59, Daniel R. Grayson wrote: Maybe now is a good
>     time to add instructions about using the standard Coq installed on
>     the system, instead of compiling the submodule. This is done with
>     BUILD_COQ=no, and can be set permanently following the instructions
>     in build/Makefile-configuration-template
>     Are we currently testing against a standard Coq, i.e., against a
>     released version?
> 
> Yes. Well, you're usually the one who updates it when a new standard 
> version comes out, and sub/coq is always pointing to the latest standard 
> version.

Well, that was before you filed this PR: 
https://github.com/UniMath/UniMath/pull/1165

**Comment by DanGrayson** [comment]
Oh, right!

**Comment by DanGrayson** [comment]
I'll test it.

**Comment by DanGrayson** [comment]
Yes, UniMath compiles with Coq 8.9.0 as provided by homebrew on the Mac.

**Comment by DanGrayson** [comment]
One problem with not building Coq in its submodule is that our ProofGeneral initialization file, UniMath/.dir-locals.el, still expects it to be there, and thus Coq fails to start.  We could customize that, too, and make it depend on Makefile and on build/Makefile-configuration, in order to fix it.

**Comment by benediktahrens** [comment]
On 04/03/2019 12:27, Daniel R. Grayson wrote:
> One problem with not building Coq in its submodule is that our 
> ProofGeneral initialization file, UniMath/.dir-locals.el, still expects 
> it to be there, and thus Coq fails to start. We could customize that, 
> too, and make it depend on Makefile and on build/Makefile-configuration, 
> in order to fix it.

Any change introduced at this stage needs rigorous testing on various 
OSes before it can be deployed.

My priority for now is to have a fail-safe setup for the UniMath school.
For its participants, the "all-inclusive" solution that we provide is 
most suitable.

**Comment by DanGrayson** [comment]
okay.

**Comment by DanGrayson** [comment]
lgtm

**Pull request merged**